python/cudf_polars/tests/expressions/test_stringfunction.py

```diff
@@ -470,3 +470,67 @@ def test_string_to_numeric_invalid(numeric_type):
 def test_string_join(ldf, ignore_nulls, delimiter):
     q = ldf.select(pl.col("a").str.join(delimiter, ignore_nulls=ignore_nulls))
     assert_gpu_result_equal(q)
+
+
+@pytest.mark.parametrize(
+    "width",
+    [
+        1,
+        0,
+        999,
+        pytest.param(
+            -1, marks=pytest.mark.xfail(reason="negative width fails before collect")
+        ),
+        pytest.param(
+            None, marks=pytest.mark.xfail(reason="None width fails before collect")
```

Comment:
There's an `assert_collect_raises` function that would be great to use so we know if this collection failure happens on the polars or cuDF side